packages/react-meteor-data/package.js

```diff
@@ -6,13 +8,17 @@ Package.describe({
   git: 'https://github.com/meteor/react-packages',
 });
 
-Package.onUse(function (api) {
+Package.onUse((api) => {
```

Comment:
I just read `Note that currently the package.js file itself is processed as Javascript/ES5. This cannot be changed.`
https://github.com/meteor/meteor/tree/master/packages/tinytest#structuring-the-test-file

Not sure what or if something will fail, but you might want to consider using ES5 here.